lib/python/treadmill_aws/sproc/garbage_collector.py

```diff
@@ -0,0 +1,53 @@
+"""Garbage collector for LDAP and IPA.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import click
+from treadmill import cli
+from treadmill import plugin_manager
+
+_LOGGER = logging.getLogger(__name__)
+_MODULE = 'treadmill_aws.garbage_collector'
```

Comment:
this is good, but this actually belongs to core, as you made it 100% generic. 

namespace to load becomes treadmill.garbage_collector, and i would add two separate modules - one for ldap, one for ipa, run them both.

Reply:
Moved to core as suggested